daq_lib.py

```diff
@@ -796,14 +796,14 @@ def center_on_click(x,y,fovx,fovy,source="screen",maglevel=0,jog=0,axisX="x",axi
     correctedX = x * daq_utils.unitScaling
   elif (axisX=="y"):
     correctedY = x * daq_utils.unitScaling
-  elif (axisX=="z"): #focus axis, 
-    #TODO: focus adjust 
+  elif (axisX=="z"): #focus axis, needs proper handling
+    correctedZ = x * daq_utils.unitScaling #TODO: focus adjust 
```

Comment:
you only needed to put in "`pass`" or "`...`" - no statement is necessary beyond these.

Reply:
the properly scaled Z value isn't being applied to the movement yet, but it's still the correct value. I need to look through the various z-axis control options to make sure I move it correctly.

Reply:
I thought your problem was with indents? that was what my comment was referring to.

by the way, the really correct way to do these kinds of transformations would be with matrices, which would allow for non-ideally aligned axes and define everything in one shot.

I know they are a pain to understand, though, so I am ok with this method for now. see this for an idea what I am talking about: https://www.meccanismocomplesso.org/en/3d-rotations-and-euler-angles-in-python/